lib/peep.rb

```diff
@@ -1,25 +1,13 @@
-require 'pg'
+require 'database_connection'
 
 class Peep
   def self.all
-    if ENV['ENVIRONMENT'] == 'test'
-      con = PG.connect(dbname: 'chitter_app_test')
-    else
-      con = PG.connect(dbname: 'chitter_app')
-    end
+    rs = DatabaseConnection.query("SELECT * FROM peeps")
 
-    rs = con.exec("SELECT * FROM peeps")
     rs.map { |peep| peep['content'] }
   end
 
   def self.create(content:)
-    if ENV['ENVIRONMENT'] == 'test'
-      con = PG.connect(dbname: 'chitter_app_test')
-    else
-      con = PG.connect(dbname: 'chitter_app')
-    end
-
-    rs = con.exec(
-      "INSERT INTO peeps (content) VALUES ('#{content}');")
+    rs = DatabaseConnection.query("INSERT INTO peeps (content) VALUES ('#{content}');")
```

Comment:
Because you are using the exec_params with your .query, should not there be a second argument, with an array of parameters? Not sure if I'm right though.

Reply:
Good question!
I believe I set up the DB connection with this method: self.query(sql_string, params = [ ])
so the second parameter is set to default? I wonder 😕
I will investigate this 🙈